lib/Github/Api/Repo.php

```diff
@@ -60,6 +60,23 @@ public function all($id = null)
     }
 
     /**
+       * List all repositories owned by the user.
+       *
+       * @link https://developer.github.com/v3/repos/#list-all-public-repositories
+       *
+       * @param int|null $id The integer ID of the last Repository that you’ve seen.
+       *
+       * @return array list of users found
+       */
+      public function owned($id = null)
```

Comment:
We should allow for parameters here instead of just the `id`. 

See a full set of parameters here: https://developer.github.com/v3/repos/#list-your-repositories